app/scripts/lib/manifestFlags.ts

```diff
@@ -0,0 +1,31 @@
+import browser from 'webextension-polyfill';
+
+export type ManifestFlags = {
+  circleci?: {
+    enabled: boolean;
+    branch?: string;
+    buildNum?: number;
+    job?: string;
+    nodeIndex?: number;
+    prNumber?: number;
+  };
+  doNotForceSentryForThisTest?: boolean;
+};
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions -- you can't extend a type, we want this to be an interface
```

Comment:
nit: I think you can do it with something like this placed in a `browser.d.ts` file in our `/types` directory.

```
// in app/scripts/lib/manifestFlags.ts
// use
browser.runtime.getManifest<ManifestFlags>()._flags

or 

browser.runtime.getManifest<ManifestFlags | undefined>()._flags
```

then in `/types/browser.d.ts`:

```
import 'webextension-polyfill';

declare module 'webextension-polyfill' {
  // Extend the existing WebExtensionManifest to include _flags
  namespace Manifest {
    interface WebExtensionManifestWithFlags<Flags>
      extends WebExtensionManifest {
      _flags: Flags;
    }
  }

  namespace Runtime {
    interface Static {
      // extend `getManifest` to allow for expecting a flags property
      getManifest<Flags = never>(): [Flags] extends [never]
        ? Manifest.WebExtensionManifest
        : Manifest.WebExtensionManifestWithFlags<Flags>;
    }
  }
}

```

You wouldn't have to make it generic, you could also hard code the types in the `/types/browser.d.ts`, or just make it return something like `Record<string, unknown>` then cast `_flags` like you already are.

If you don't want to include ` | undefined` in the generic you can set the `_flags: Flags` type in `browser.d.ts` to `_flags?: Flags` instead.

Again, just a nit.